src/core_init_atmosphere/mpas_init_atm_cases.F

```diff
@@ -2907,6 +2907,12 @@ subroutine init_atm_case_gfs(block, mesh, nCells, nEdges, nVertLevels, fg, state
             hs(iCell) = 0.
             if(ter(iCell) .ne. 0.) then
                do j = 1,nEdgesOnCell(iCell)
+
+                  ! For smoothing at the edge of the mesh, effectively ignore non-existent neighbors
```

Comment:
I know its getting a bit into the 'what' side of comments, but it might be worth adding the method for ignoring non-existent neighbors i.e. setting them equal to the edge cell.
```
! For smoothing at the edge of the mesh, effectively ignore non-existent neighbors by setting them equal
! to the edge cell
```

Reply:
New comments have been added.